src/javascripts/components/mushroomList/mushroomList.js

```diff
@@ -93,6 +102,18 @@ const mycoMushroomController = (e) => {
   }
 };
 
+const showImages = (mushrooms) => {
+  mushrooms.forEach((mushroom) => {
+    if (!mushroom.image) return;
+
+    const storageRef = firebase.storage().ref(`mushrooms/${mushroom.image}`);
+
+    storageRef.getDownloadURL().then((url) => {
+      $(`#${mushroom.id} .mush-img`).attr('src', url);
+    });
+  });
+};
```

Comment:
similar here: instead of `put`-ing the image, we're looping over all the mushrooms, creating a reference to where they're stored on firebase, getting the "downloadURL", and once we have gotten that setting the appropriate img src to the download url.